assets/js/vue-apps/components/grant-filters.vue

```diff
@@ -0,0 +1,103 @@
+<script>
+import FacetGroup from './facet-group.vue';
+import FacetDisclose from './facet-disclose.vue';
+import FacetChoice from './facet-choice.vue';
+import FacetSelect from './facet-select.vue';
+
+import partition from 'lodash/partition';
+
+export default {
+    components: { FacetGroup, FacetDisclose, FacetChoice, FacetSelect },
+    props: ['facets', 'filters', 'status'],
+    computed: {
+        programmes() {
+            const [featured, other] = partition(this.facets.grantProgramme, programme => {
+                return ['awards for all'].some(allowed => programme.value.toLowerCase().indexOf(allowed) !== -1);
```

Comment:
Fine for now – possibly with the programme enhancements recently added to the API we could do that there (eg. add a `isFeatured` flag?).